mindsdb/interfaces/storage/fs.py

```diff
@@ -314,3 +343,137 @@ def __call__(self, resource_id: int):
             sync=self.sync,
             resource_id=resource_id
         )
+
+
+from .json import get_json_storage
+
+class ModelStorage:
+    def __init__(self, company_id, predictor_id):
```

Comment:
add docs for class description and each public method too https://google.github.io/styleguide/pyguide.html#s3.8-comments-and-docstrings